custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html

```diff
@@ -1,43 +1,189 @@
 {% load staticfiles %}
 <div class="sideNavigation" id="nav-menu">
-    <img src="{% static "icds_reports/mobile/images/white-close-icon.png" %}" class="close-icon" ng-click="closeMenu()">
-    <p class="navigation-font" ng-bind="userFullName" ng-if="userFullName"></p>
-    <p class="navigation-font mb-5" ng-bind="userUsername"></p>
-    <a class="menu-options" href="{$ goToStep('', $location.search()) $}" ng-click="closeMenu()">
-        <img src="{% static "icds_reports/mobile/images/program-summary.png" %}" class="mr-3">
-        <p class="navigation-font mb-0">Program Summary</p>
-    </a>
-    <div class="menu-options">
-        <img src="{% static "icds_reports/mobile/images/mother.png" %}" class="mr-3">
-        <p class="navigation-font mb-0">Maternal and Child Nutrition</p>
+    <img src="{% static 'icds_reports/mobile/images/white-close-icon.png' %}" class="close-icon" ng-click="closeMenu()">
+    <p class="navigation-font pl-3 pr-3 mb-0" ng-bind="userFullName" ng-if="userFullName"></p>
+    <p class="navigation-font pl-3 pr-3 mb-4" ng-bind="userUsername"></p>
+    <div style="overflow: auto; height: calc(100vh - 100px)">
```

Comment:
is it required that this be an inline style declaration?

Reply:
I will update it